utils/utils.go

```diff
@@ -367,3 +369,56 @@ func GetLocalIP() (string, error) {
 func GetFuncName(fn interface{}) string {
 	return runtime.FuncForPC(reflect.ValueOf(fn).Pointer()).Name()
 }
+
+func WritePidToFile(pid int, path string) error {
```

Comment:
I'd like to keep utils within the packages that use them, until they become usefull elsewhere. Then they can be moved into the common utils package. To better organize code, add these in a `utils.go` file in the `daemon` package.